### ISSUE TITLE
Deprecation warnings with rubocop 1.65

### ISSUE THREAD

**Issue by klyonrad** [prontolabs/pronto-rubocop#90]
Since rubocop release 1.65 https://github.com/rubocop/rubocop/releases/tag/v1.65.0

executing pronto shows many many deprecation warnings from rubocop. See also https://github.com/rubocop/rubocop/pull/13032

```
pronto-rubocop-0.11.5/lib/pronto/rubocop/patch_cop.rb:34: warning: `Cop.all` is deprecated. Use `Registry.all` instead.
pronto-rubocop-0.11.5/lib/pronto/rubocop/patch_cop.rb:62: warning: `inspect_file` is deprecated. Use `investigate` instead.
```

anything special to think about or would this be a simpel replacement of the used methods?

**Comment by amomchilov** [comment]
Nope. The method just forwarded anyway, you can just add in the `.class`

**Comment by jeantristan** [comment]
same error here.
I did not get what I had to do to fix the error

**Comment by ashkulz** [comment]
fixed via #91

**Comment by Jakanapes** [comment]
Is there a release coming soon?

**Comment by MoawazAyub** [comment]
Is there a release coming soon? with the fix #91
